tsrc/workspace/syncer.py

```diff
@@ -12,9 +12,10 @@ class BadBranches(tsrc.Error):
 
 
 class Syncer(tsrc.executor.Task[tsrc.Repo]):
-    def __init__(self, workspace_path: Path) -> None:
+    def __init__(self, workspace_path: Path, *, force: bool = True) -> None:
```

Comment:
Force is a "dangerous" operation, probably should go with a false for default

Reply:
Good catch :)